tomls/markets/common/settlement-settings.toml

```diff
@@ -3,7 +3,7 @@
 [setting.pythFeedUrl]
 
 [setting.settlementReward]
-defaultValue = "0"
+defaultValue = "0.1"
```

Comment:
Move my comment from #71 

Because settlement settings are not updatable but append only we cannot just change settings. May I suggest removing this file entirely and inline settings into the addSettlementStrategy invoke instead? Because once we add it we cannot update or remove it. Any change will require adding a new strategy and disabling the old one.